docs/Custom-brimcap-load-Config.md

```diff
@@ -0,0 +1,434 @@
+# Custom `brimcap load` Configuration
+
+- [Summary](#summary)
+- [Custom Zeek/Suricata Analyzers](#custom-zeeksuricata-analyzers)
+  * [Background](#background)
+  * [Base Zeek/Suricata Installation](#base-zeeksuricata-installation)
+  * [Example Configuration](#example-configuration)
+- [Custom NetFlow Analyzer](#custom-netflow-analyzer)
+  * [Background](#background-1)
+  * [Base nfdump Installation](#base-nfdump-installation)
+  * [Example Configuration](#example-configuration-1)
+- [Contact us!](#contact-us)
+
+# Summary
+
+When used out-of-the-box in a default configuration, `brimcap load` invokes
+Zeek and Suricata binaries that are bundled with Brimcap to generate
+richly-typed summary logs from pcap files. However, one of Brimcap's strengths
+is its ability to invoke any arbitrary combination of such "analyzers" that
+generate logs from pcaps. These analyzers could be alternate Zeek or Suricata
+installations that you've customized or other pcap-processing tools of your
+choosing.
+
+This article describes such example custom configurations. The first section
+focuses on recreating Brimcap configurations equivalent to those created with
+the customization options in Brim app release `v0.24.0` and older. The second
+section builds on the approach to create a custom Brimcap configuration that
+generates NetFlow records from pcaps.
+
+# Custom Zeek/Suricata Analyzers
+
+## Background
+
+Brim release `v0.25.0` bundles Zeek and Suricata binaries differently than
+`v0.24.0`. A brief summary:
+
+* **`v0.24.0` and older** - The "backend" `zqd` process that ran behind the
+Brim app handled the launching of Zeek and Suricata binaries each time a pcap
+was imported. The launch of these processes was done via "runner" scripts that
+ensured appropriate pathnames and environment variables were set for them to
+run successfully. A [Zeek Customization](https://github.com/brimdata/brim/wiki/Zeek-Customization)
+article described how an alternate "Zeek Runner" script could be
+created/invoked such that Brim would launch Zeek from an install different than
+the one bundled with Brim. A similar option existed to invoke an alternate
+"Suricata Runner" (as well as a "Suricata Updater" for updating
+[rules](https://suricata.readthedocs.io/en/suricata-6.0.0/rules/)) however
```

Comment:
Maybe replace `suricata-6.0.0` with `latest` in all the Suricata documentation URLs?